calculations.py

```diff
@@ -0,0 +1,118 @@
+def generate_unique_items_list(data):
+    items_id_column_index = 0
+    items_id_list = []
+
+    for line in data:
+        if not line[items_id_column_index] in items_id_list:
+            items_id_list.append(line[items_id_column_index])
+
+    return items_id_list
```

Comment:
It's better to use set, not list, so that you don't need to check if value was already added. You can also use a comprehension instead of the loop